openedx/core/djangoapps/programs/models.py

```diff
@@ -20,6 +20,12 @@ class ProgramsApiConfig(ConfigurationModel):
     public_service_url = URLField(verbose_name=_("Public Service URL"))
     api_version_number = IntegerField(verbose_name=_("API Version"))
     enable_student_dashboard = BooleanField(verbose_name=_("Enable Student Dashboard Displays"))
+    cache_ttl = IntegerField(
+        verbose_name=_("Cache Time To Live"), default=0, help_text=_("Time is specified in seconds")
+    )
+    enable_cache = BooleanField(verbose_name=_("Enable Cache For Programs Data on Student Dashboard."))
```

Comment:
It seems like we could make this simpler by simply treating ttl=0 as cache disabled, then we would not need a separate boolean toggle.